src/lib/util/Utils.ts

```diff
@@ -13,4 +12,12 @@ const stringArrayTypeGuard = (object): object is string[] => {
 	return !object.some(entry => typeof entry !== 'string');
 }
 
-export { deepCopy, stringArrayTypeGuard };
+const booleanTypeGuard = (object): object is boolean => {
```

Comment:
Currently, `Config.ts` is the only place these new type guards are used. I considered defining them in that file instead of here, but it seems preferable to define them here for consistency's sake.